.github/workflows/release.yaml

```diff
@@ -13,17 +13,32 @@ jobs:
 
   release:
     needs: check
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - [ubuntu-latest]  # github hosted runners are amd64
+            # Ubuntu_ARM64_4C_16G_01 is the github-hosted arm64 runner we have access to.
+            # We prefer the github runners because they are smaller machines and save resources.
+            # If we have issues with it, we can switch to the larger and more numerous self-hosted options:
+            # - runs-on: [self-hosted, jammy, ARM64]
+          - [Ubuntu_ARM64_4C_16G_01]
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
       - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
         uses: canonical/setup-lxd@v0.1.1
+
+        # NOTE: this action only supports a single charm file output https://github.com/canonical/charming-actions/blob/2.4.0/src/services/charmcraft/charmcraft.ts#L193-L194
+        # This is fine right now because the charm only has a single build per architecture,
+        # so only one will be built here.
       - name: Upload charm to charmhub
         uses: canonical/charming-actions/upload-charm@2.4.0
```

Comment:
it's there any concern bumping it to the latest (as of now: 2.6.2) ?

Reply:
Updated